src/content/developers/docs/scaling/plasma/index.md

```diff
@@ -18,9 +18,36 @@ You should have a good understanding of all the foundational topics and a high-l
+
+## How does Plasma differ from sidechains and sharding? 
+
+Plasma, sidechains, and sharding all follow a similar pattern where a collection of "child chains" rely on a "main chain" for consensus. At intervals, cryptographic proofs of transaction validity (hashes) are submitted to the main chain to prevent fraudulent activity on child chains. 
+
+However, the three scaling solutions have slight differences. Here is a quick comparison of Plasma chains, sidechains, and shard chains:
+
+### Plasma vs sidechains  
+
+A [sidechain](https://ethereum.org/en/developers/docs/scaling/sidechains/) is a blockchain operating independently but connects to the main blockchain (Ethereum Mainnet) via a two-way bridge. [Bridges](https://ethereum.org/en/bridges/) allow users to exchange tokens between both blockchains, so they can transact on the sidechain—reducing congestion on the base layer and improving scalability. 
+With sidechains, there's the risk of losing your funds. Sidechains use a separate consensus mechanism and, ergo, cannot rely on the security of Ethereum Mainnet (Layer 1).
+
+Conversely, Plasma chains are somewhat safer than sidechains because they derive their security from Ethereum's base layer. Like sidechains, Plasma chains can have a  different consensus protocol. 
+
+The difference is that Plasma publishes Merkle roots for each block on the Ethereum main chain. Block roots are small pieces of information we can use to verify information about transactions that happen on a Plasma chain. If an attack happens on a Plasma chain, users can safely exit to the main chain and withdraw their funds using the appropriate proofs. 
+
+A simple way to differentiate between Plasma and sidechains is that Plasma chains are non-custodial, while sidechains are custodial. 
+
+### Plasma vs sharding
+
+Both Plasma chains and [shard chains](https://ethereum.org/en/upgrades/shard-chains/) periodically publish cryptographic proofs on the main chain. However, both have different security properties. 
+Shard chains are tightly coupled to the main chain, but plasma chains are not. Even if one shard block is invalid, the entire chain will reorganize and discard the invalid block. Thus, shard chains can be as safe as the main chain. 
+
+While Plasma chains can benefit from Layer 1 security, they are not as tightly linked to the main chain as shard chains. It's possible to commit an invalid Plasma block header to the main chain since the larger network has little information about the state of the Plasma chain. 
+
+With sharding, the main chain knows how sharded sub-chains should operate and can reject an invalid shard transition. The upside to Plasma is the effects of a malicious activity can be restricted to the child chain, while the effect on malicious activity on shard chains would affect the entire network. 
```

Comment:
Overall a little hesitant to publish anything more on shard chains for the moment, given danksharding updates, and  especially hesitant in the active voice too, where it gives the impression this is live on Ethereum today.

I think we can save this still though, to fill out the comparison. I would consider rephrasing this a bit to be slightly more generalized, and discuss the general concept of a shard of data, and how its validity can be enforced by Mainnet, compared to plasma where it is not.

Also, afaik no need to capitalize "plasma." (and link needs domain/language trimmed out)

Reply:
So, I reworked the content here and took out the last paragraph. Let me know if the changes are good enough.